routes/idp.js

```diff
@@ -35,15 +36,7 @@ module.exports = function (log, serverPublicKey) {
         // FOR DEV PURPOSES ONLY
         reply(
           {
-            keys: [
-              {
-                kid: "dev-1",
-                use: "sig",
-                kty: "RSA",
-                n: b64(bigint(serverPublicKey.n).toBuffer()),
-                e: b64(bigint(serverPublicKey.e).toBuffer())
-              }
-            ]
+            keys: [ serverPublicKey ]
```

Comment:
Maybe I'm not following the code here, but in definition of `browseridKey` above we access e.g. `serverPublicKey.jwk.algorithm`, and here we expose `serverPublicKey` directly.  Should we be exposing `serverPublicKey.jwk` instead?